hack/helm/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
Why not use the default SA in the `hnc-system` directory? That would eliminate this and (I think) the cluster role binding.

Reply:
that would work but my understanding of the hnc docs is we should/must apply the labels first before installing the controller right? If that's the case we need to pull some things out of band specifically
- install the sa / rolebinging
- run the job that adds the label
- now install the controller et al

Reply:
Ah, I see. In that case, can we call it something like `hnc-installer` so that it's clear what this is for? Ditto for the cluster role binding.

Also, if it's essentially a mandatory part of installing HNC, why are we giving users the option to disable it?

As a side note, you actually can apply the labels _after_ installing HNC, but that will mean that there's a period of time during which the namespaces are not actually excluded. Worse, while HNC is starting up, it may forbid changes to the namespaces, so it could take a while to add the labels. So it's definitely _better_ to label them before installing HNC.